### ISSUE TITLE
ensureSignedIn plugin is not working in android-device app.

### ISSUE THREAD

**Issue by candicom** [meteor-useraccounts/core#446]
Router.plugin('ensureSignedIn', {
    only: ['set']
});

AccountsTemplates.configureRoute('signIn', {
  template: 'accounts'
});

And 

meteor run android-device -p 3000

And Application shows up on my android mobile phone.

But the link to /set is just appeared without login.

So I guess ensureSignedIn plugin is not working in cordova app.

And I changed to

Router.route('set', {
  name: 'set',
  onBeforeAction: function() {
    if (! Meteor.userId()) {
      Router.go('/sign-in');
    } else {
      this.next();
    }
  }
});

AccountsTemplates.configureRoute('signIn', {
  template: 'accounts'
});

But also It has an error.

I20150628-11:14:24.560(9) (android:http://meteor.local/packages/useraccounts_core.js:988) Uncaught Error: changePwd route configured but enablePasswordChange set to false!

Finally, I set these... I could see on my android device.

Router.route('sign-in', {
  name: 'signIn',
  template: 'accounts'
});

<template name="accounts">

    {{>mainMenu}}

    {{>atForm}}

</template>

//AccountsTemplates.configureRoute('signIn', {
//  template: 'accounts'
//}); <- I removed it.

So in conclusion, AccountsTemplates.configureRoute is not working on android-device cordova app.
ensureSignedIn also doesn't work. It also some kind of Route.

**Comment by splendido** [comment]
I'd say this is likely to be an issue with the [iron:router package](https://github.com/iron-meteor/iron-router)...
Have you checked among the issues there?

**Comment by elie222** [comment]
I also get `Uncaught Error: changePwd route configured but enablePasswordChange set to false!` when running my app with `meteor run android-device --mobile-server example.com`.

**Comment by elie222** [comment]
Here's a fuller log:

```
=> App running at: http://localhost:3000/
I20150831-22:45:50.358(3) (android:http://meteor.local/packages/useraccounts_iron-routing.js:157) Uncaught Error: changePwd route configured but enablePasswordChange set to false!
I20150831-22:45:50.911(3) (android:http://meteor.local/packages/iron_core.js:89) pathFor couldn't find a route named "home"
I20150831-22:45:50.920(3) (android:http://meteor.local/packages/iron_core.js:89) pathFor couldn't find a route named "atSignIn"
```

**Comment by elie222** [comment]
For me it suddenly started working. Not sure why. Had to download the app from the server to start working maybe?
